daktari/file_utils.py

```diff
@@ -11,10 +11,7 @@ def is_ascii(path: str) -> bool:
 
 def file_exists(path: str) -> bool:
     testing_file = Path(path)
```

Comment:
Looking at the docs, there's a method `Path.expanduser()` https://docs.python.org/3/library/pathlib.html?highlight=path#pathlib.Path.expanduser, looks like the only difference from `os.path.expanduser` is in the error handling.

Is it nicer to do 
```suggestion
    testing_file = Path(path).expanduser()
```
then you don't have to import `os.path.expanduser` and remember to use it each time you use these utils in `checks/files.py`?

Reply:
That's useful to know, thanks. I think it might actually be simpler the way it is already, because calling code quite often  needs the expanded path when calling other functions (especially library functions we can't add this expansion to). For example, we might check for a file's existence before later opening it with `open()`.

I think it's safer too not to assume we always want '~' expansion - this function doesn't know if the input has already been expanded. It'd be asking for trouble to call a file '~' but it's not impossible...

Reply:
Had a bit of a play, and it looks like the `Path.expanduser()` is pretty smart. e.g. I created a file `~/testing/dir~dir/~/test.py`, which runs 
`from pathlib import Path `
`print('working')`
`print(Path.cwd())`
and running it from inside that gross directory correctly expands only the first `~`
`working`
`/Users/matthewkandiah/testing/dir~dir/~`
so maybe that's not a real issue.

Reply:
As for whether it's neater on the whole, I'm happy to defer to you on that, I'm not super familiar with the daktari checks. It still seems to me like it would be nicer to expand the path in the util functions (assuming I'm correct and it's safe, even in the weird edge cases), but I'm happy for you to leave it if you disagree, I think it's a small difference.

Reply:
Yeah, looks like `expanduser()` will do something useful in pretty much all cases except when there's a file or directory called '~' in the current directory. I realise that point's a bit nit-picky!

But it's still true that library functions don't do this expansion, so we'd have an inconsistency where code elsewhere sometimes needs to expand the path and sometimes doesn't. You could call `file_exists()` and find that a file exists at `~/test.txt`, then fail to open that same path with `open()`, for example. So I think although it's a bit sad that we have to remember to call it in a lot of places, we do avoid a potentially confusing inconsistency.